staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -817,7 +817,7 @@ func (r *Reflector) LastSyncResourceVersion() string {
 	return r.lastSyncResourceVersion
 }
 
-func (r *Reflector) setLastSyncResourceVersion(v string) {
+func (r *Reflector) SetLastSyncResourceVersion(v string) {
```

Comment:
exposing this significantly increases the risk something will set it incorrectly and make the reflector behave incorrectly... I really don't think we should do that.